src/net_processing.h

```diff
@@ -42,6 +42,9 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
                                              CTxMemPool& pool, bool ignore_incoming_txs);
     virtual ~PeerManager() { }
 
+    /** Attempt to manually fetch block from a given node. */
+    virtual bool FetchBlock(const NodeId nodeid, const CBlockIndex& pindex) = 0;
```

Comment:
Remove `const` from the pass-by-value argument:

```suggestion
    virtual bool FetchBlock(NodeId nodeid, const CBlockIndex& pindex) = 0;
```